ci/test-upgrade/run.sh

```diff
@@ -40,30 +40,16 @@ make ci-testupgrade-create-cluster
 
 # TODO(sreis): Add a new sanity check on the base cluster here before running
 # the upgrade. https://github.com/opstrace/opstrace/issues/591
-
 make ci-testupgrade-upgrade-cluster
 
 # Define default for OPSTRACE_INSTANCE_DNS_NAME.
 export OPSTRACE_INSTANCE_DNS_NAME="${OPSTRACE_CLUSTER_NAME}.opstrace.io"
 
-# This step is required for the create_tenant_and_use_custom_authn_token in the
-# test-remote test suite. This step sets up the tenant keys in the opstrace
-# instance and exports the required env vars:
-# - TENANT_RND_NAME_FOR_TESTING_ADD_TENANT
-# - TENANT_RND_AUTHTOKEN
-#
-make ci-testupgrade-set-up-test-tenant
-export TENANT_RND_NAME_FOR_TESTING_ADD_TENANT=$(cat tenant-rnd-name)
-export TENANT_RND_AUTHTOKEN=$(cat tenant-rnd-auth-token-from-custom-keypair)
-
-#
+
 # TODO(sreis): remove this step when the OPSTRACE_CLI_VERSION_FROM is bumped
-#
 make ci-testupgrade-wait-for-loki-ring
 
-make test-remote
-make test-remote-ui
-make test-browser
+# This runs the bulk of the tests against the Opstrace instance, also invoked
+# from the upgrade test pipeline and therefore in its own file.
+source ci/test-core.sh
```

Comment:
Until recently there was no need to do any sort of docker in docker inception so the ci/test-upgrade/run.sh script was kept simple.

This will most fail when the kubectl commands try to refresh the AWS/GCP kubectl credentials because the script (ci/test-upgrades/run.sh) is not running in a container. The credentials refresh will fail to find the aws/gcloud binaries.

Reply:
Ah right, thanks for the reminder about the kubectl topic.

Yeah OK then I will transition to running this in the ci container :shrug: -- keeping the 'test code in sync' is important, and keeping symmetry across how build steps are invoked is also an advantage.